### PR TITLE
Update Alpine Version and Busybox Version

### PR DESCRIPTION
**Pull request by Jason-Clark-FG**
Hi @lipanski, thanks for the great little container that I just discovered! I wondered if I could update it, so I gave it a shot. I'm not 100% I selected the best options for the `.config` file, so feel free to make fixes! I loaded the config up in Alpine container and ran `make oldconfig` and selected the changes I thought that would keep it lightweight like you had it!

You can remove the `action` if you don't want to use that.

Thanks again! :)

### CODE REVIEW

File: .config [hunk 1]
```diff
@@ -59,7 +59,7 @@ CONFIG_EXTRA_LDLIBS=""
 #
 # Installation Options ("make install" behavior)
 #
-# CONFIG_INSTALL_APPLET_SYMLINKS is not set
+CONFIG_INSTALL_APPLET_SYMLINKS=y
```
Comment (abitrolly) [inline .config:62]: Symlinks are not needed for the container with the single process.

**Comment by lipanski** [comment]
Thanks for the PR, I'll have a closer look over the next days :100:

File: .config (hunk 1 above)
Comment (Jason-Clark-FG) [reply .config]: Thanks @abitrolly I commented that one out then (and added the "is not set")

**Comment by lipanski** [comment]
Released in [2.2.0](https://hub.docker.com/r/lipanski/docker-static-website/tags)

I picked this up in https://github.com/lipanski/docker-static-website/commit/1c19f7938f8b6fcdf42857913aff00116a4dd6e5 and added some changes

Thanks for your contribution :100:

**Comment by abitrolly** [comment]
Thanks for the update! :D

Do you know why the new image is 2kB larger?

image

**Comment by lipanski** [comment]
> Do you know why the new image is 2kB larger?

@abitrolly I guess bugfixes etc. - you can see the changes [here](https://busybox.net/)

**Comment by abitrolly** [comment]
@lipanski by the way, https://hub.docker.com/r/lipanski/docker-static-website/tags still points to an old image.

@Jason-Clark-FG your image with old version in `busybox` is indeed smaller.
```
docker.io/lipanski/docker-static-website                    2.2.0                1d6fcaa24fde  4 days ago     172 kB
docker.io/lipanski/docker-static-website                    latest               9b08b9509c32  15 months ago  168 kB
ghcr.io/jason-clark-fg/docker-static-site                   main                 26b0950179ac  3 weeks ago    114 kB                                                 /0,2s
```
Looks like binaries uploads are no automatic. At least I see the issue had already been reported for `1.3.1` version
https://bugs.busybox.net/show_bug.cgi?id=14286. I wonder which command is used to build them?

**Comment by lipanski** [comment]
@lipanski fixed, `latest` now points to `2.2.0`

regarding the size of those pre-compiled binaries, I'd be curious if they provide the same feature set. my intention was to preserve certain features of the HTTP server (basic auth, error pages, gzip, etags), which is why I didn't aggressively strip them out from the `.config`. if you want the smallest possible image, you can look into disabling all of these but I don't intend to do that in here.

**Comment by abitrolly** [comment]
Maybe the binary is smaller because there is no code for other applets? Only the statically linked core.

**Comment by lipanski** [comment]
FYI @Jason-Clark-FG we ended up adding your suggestion in https://github.com/lipanski/docker-static-website/pull/25 and released it as 2.3.0 :tada:

**Pull request closed without merging**